### PR TITLE
Add representative code to contested claimant

### LINKED ISSUE #9826: [IDT] Contested claimants missing representatives
Examples, 
vacols_id: 2695328, 3366989, 2469831, 2811717

```
SELECT * FROM REP WHERE REPKEY='2811717'
```
returns only one record that contains information about the contested claimant and their rep in the `repso` field

`repso` field contains a letter that stands for a service organization https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/vacols/case.rb#L46

AC: return representative for contested claimants for IDT

### PR DESCRIPTION
**Pull request by monfresh**
**Why**: IDT needs this info

Resolves #9826

### CODE REVIEW

File: app/models/vacols/representative.rb [hunk 1]
```diff
@@ -152,7 +152,10 @@ def as_claimant
       middle_name: repmi,
       last_name: replast,
       name_suffix: repsuf,
-      address: get_address_from_rep_entry(self)
+      address: get_address_from_rep_entry(self),
+      representative: {
+        code: repso
```
Comment (aroltsch) [inline app/models/vacols/representative.rb]: Is Paul okay with us just sending the code instead of a human readable name?

Comment (aroltsch) [reply app/models/vacols/representative.rb]: https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/vacols/case.rb#L46

File: app/models/vacols/representative.rb [hunk 2]
```diff
@@ -146,13 +146,19 @@ def as_claimant
...
       type: type,
       first_name: repfirst,
       middle_name: repmi,
       last_name: replast,
       name_suffix: repsuf,
-      address: get_address_from_rep_entry(self)
+      address: get_address_from_rep_entry(self),
+      representative: {
+        code: code,
+        name: VACOLS::Case::REPRESENTATIVES[code][:full_name]
```
Comment (aroltsch) [inline app/models/vacols/representative.rb]: I think code could be blank in some cases.

Comment (monfresh) [reply app/models/vacols/representative.rb]: Could it be blank as in an empty string? If it's `nil`, that's already covered. I will update to handle the missing letter scenario.

Comment (monfresh) [reply app/models/vacols/representative.rb]: I updated it to use `dig`, which will handle all nonexistent keys.

**Pull request merged**